jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1474,6 +1474,10 @@ properties:
             type: number
             fits_keyword: GS_DEC
             blend_table: True
+          gs_pa:
+            title: guide star V3 position angle
+            fits_keyword: GS_PA
```

Comment:
JWSTKD-422 decided to make the name of this new keyword "GS_V3_PA", so should update here to be consistent. Also check the datamodel named implemented in the Keyword Dictionary.

Reply:
✅ but not sure what the "check the datamodel..." statement means.

Reply:
> ✅ but not sure what the "check the datamodel..." statement means.

Sorry, should've been more specific. The name they chose for the meta attribute for this keyword, which in the KWD is simply called "Datamodel Name". According to the JWSTKD ticket, it looks like it's "gs_v3_pa_science"